CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
 - [#169](https://github.com/SuperGoodSoft/solidus_taxjar/pull/169) Add basic backfill transaction functionality
 - [#181](https://github.com/SuperGoodSoft/solidus_taxjar/pull/181) Take all non-tax adjustment types into account when calculating a line item's discount
 - [#182](https://github.com/SuperGoodSoft/solidus_taxjar/pull/182) Automatically create default Tax Rate
+- [#183](https://github.com/SuperGoodSoft/solidus_taxjar/pull/183) Allow admin to use TaxJar tax codes
```

Comment:
Could we re-phrase this to be a bit more end-user friendly. Maybe something like `Add ability to fetch tax codes from TaxJar and assign to tax categories in admin`